surface_potential_analysis/kernel/kernel.py

```diff
@@ -510,37 +555,24 @@ def get_noise_operators_isotropic(
         pad_ft_points(kernel["data"], (n,), (0,)),
         norm="forward",
     )
```

Comment:
why is coefficients obtained via inverse fft but not fft of beta?

Reply:
This is just to match the convention used in the original paper